### PR TITLE
[prometheus-ipmi-exporter] Add ipmi-exporter chart

### PR DESCRIPTION
**Pull request by lexfrei**
#### What this PR does / why we need it
- Adds ipmi-exporter initial chart

#### Which issue this PR fixes

- fixes https://github.com/prometheus-community/ipmi_exporter/issues/109

#### Special notes for your reviewer
сс @bitfehler

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-ipmi-exporter/Chart.yaml
Comment (zanhsieh) [inline charts/prometheus-ipmi-exporter/Chart.yaml:9]: @lexfrei 
Thanks for the contribution. Would you mind to add some maintainer(s), e.g. yourself? A chart without maintainer(s) would be hard to keep it up.

**Review by zanhsieh — CHANGES_REQUESTED** [review]

File: charts/prometheus-ipmi-exporter/Chart.yaml
Comment (lexfrei) [reply charts/prometheus-ipmi-exporter/Chart.yaml:9]: Im fine to maintain this chart. Please, give me a link to some guides about how to do this in prometheus community :)

File: charts/prometheus-ipmi-exporter/templates/deployment.yaml
Comment (zanhsieh) [inline charts/prometheus-ipmi-exporter/templates/deployment.yaml:12]:
```suggestion
  selector:
    matchLabels:
      {{- include "prometheus-ipmi-exporter.selectorLabels" . | nindent 6 }}
```

**Review by zanhsieh — CHANGES_REQUESTED** [review]

**Review by zanhsieh — APPROVED** [review]

**Comment by lexfrei** [comment]
@zanhsieh do we need to do something to merge this PR?

**Comment by bitfehler** [comment]
Just wanted to say thanks and that I will link to this in the ipmi exporter README once it landed! Cheers!

**Comment by zanhsieh** [comment]
Maybe ask @monotek to see if he can merge it. I can't merge it.

**Comment by adyekjaer** [comment]
@zeritti Any chance this can be merged ?

**Comment by zeritti** [comment]
> @zeritti Any chance this can be merged ?

I am sorry, I cannot help. An approval by @prometheus-community/helm-charts-admins is required.

**Review by monotek — APPROVED** [review]

**Pull request merged**
